xmtp_mls/src/client.rs

```diff
@@ -617,6 +619,63 @@ mod tests {
         assert_eq!(decrypted, to_encrypt);
     }
 
+    #[tokio::test]
+    async fn test_add_remove() {
+        let amal = ClientBuilder::new_test_client(generate_local_wallet().into()).await;
+        let bola = ClientBuilder::new_test_client(generate_local_wallet().into()).await;
+        bola.register_identity().await.unwrap();
+
+        // Create a group and invite bola
+        let amal_group = amal.create_group().unwrap();
+        amal_group
+            .add_members_by_installation_id(vec![bola.installation_public_key()])
+            .await
+            .unwrap();
+        assert_eq!(amal_group.members().unwrap().len(), 2);
+
+        // Now remove bola
+        amal_group
+            .remove_members_by_installation_id(vec![bola.installation_public_key()])
+            .await
+            .unwrap();
+        assert_eq!(amal_group.members().unwrap().len(), 1);
+
+        // See if Bola can see that they were added to the group
+        bola.sync_welcomes().await.unwrap();
+        let bola_groups = bola.find_groups(None, None, None, None).unwrap();
+        assert_eq!(bola_groups.len(), 1);
```

Comment:
? - doesn't need to be answered here, but is there any mechanism for `bola` to see they've been removed from the group?
I read this as:
- bola was added to one (1) group;
- bola was then removed from said group;
- bola sees that they're in one (1) group ...?

Reply:
MLS has a way, but we aren't currently bubbling it up. That's coming next